src/app/modules/command/components/limit-command/limit-command.component.html

```diff
@@ -3,7 +3,7 @@
     <div nz-row [nzGutter]="12">
       <div nz-col [nzSm]="12" [nzXs]="24">
         <nz-form-item class="compact">
-          <nz-form-control [nzErrorTip]="quantityError">
+          <nz-form-control [nzErrorTip]="quantityError" [nzValidateStatus]="form.get('quantity')?.invalid ? 'error' : ''">
```

Comment:
```suggestion
          <nz-form-control [nzErrorTip]="quantityError" [nzValidateStatus]="form.controls.quantity">
```
nzValidateStatus принимает FormControl - нет необходимости явно указывать статус 'error'